README.md

```diff
@@ -34,6 +34,11 @@ dnf install flex bison readline-devel zlib-devel openssl-devel \
   libseccomp-devel perl clang cmake postgresql postgresql-contrib protobuf-compiler \
   protobuf-devel
 ```
+* On Arch based systems, these packages are needed:
+```bash
+pacman -S base-devel libtool readline zlib flex bison libseccomp openssl clang \
+pkg-config postgresql-libs cmake postgresql protobuf
```

Comment:
Not an expert by any means, but my reading is that `pkg-config` has since been replaced by `pkgconf`, which is already provided by `base-devel`. Is there a reason to include `pkg-config` specifically?

Reply:
I hadn't realised that it'd been replaced by pkgconfig, I can remove that from the command alongside flex, bison and libtool which are also found in base-devel.

Reply:
unresolving; `pkg-config` is still there ­— otherwise looks good.

Reply:
oops, missed that one when removing the others, fixed in 70e29527ae4d7d78574eb457774d66f9cc6c1436